test/study-add.js

```diff
@@ -86,6 +85,8 @@ describe('Add a new study', function studyAdd() {
             .setValue('input[name=label]', sampleData.label)
             .setValue('input[name=hrrc_title]', sampleData.title)
             .setValue('input[name=study_dir_name]', sampleData.archiveDirectory)
+            .click('//*[@id="frmAdd"]/table[1]/tbody/tr[6]/td[2]/input')
```

Comment:
It works as I tested it. But I don't understand the syntax, could you help me understand it , maybe with a reference?

Reply:
Since the input box didn't have an id, I used xPath of the element for selection. I have used chrome to get the xPath of the checkbox.

Reply:
Thanks for the info!